app/helpers/articles_helper.rb

```diff
@@ -17,6 +19,10 @@ def has_vid?(article)
       article.comments_blob.include?("youtube")
   end
 
+  def has_long_markdown?(article)
```

Comment:
**nitpick (non-blocking):** It's somewhat unidiomatic to prefix predicate methods with `has_` or `is_` in Ruby (see [Predicate Method Prefix](https://github.com/rubocop-hq/ruby-style-guide#predicate-methods-prefix) in the Rubocop styleguide). 

```suggestion
  def long_markdown?(article)
```